test/config-format/typescript/typescript.test.js

```diff
@@ -4,7 +4,7 @@ const { stat } = require('fs');
 const { resolve } = require('path');
 
 describe('webpack cli', () => {
-    it(
+    it.skip(
```

Comment:
why skip?

Reply:
Not working with `nyc` 🤷🏻‍♂️ , need a fix.

Reply:
I don't understand why we need nyc when jest has it inbuilt?

/cc @evilebottnawi 

Maybe we should remove it and use `jest --coverage`

Reply:
https://github.com/facebook/jest/issues/3190

Reply:
ahh spawned process mess, we need to get to this

Reply:
Yep spawned process